keanu-python/keanu/algorithm/sampling.py

```diff
@@ -81,9 +80,12 @@ class NUTSSampler(PosteriorSamplingAlgorithm):
 
     def __init__(self,
                  adapt_count: int = None,
+                 adapt_step_size_enabled: bool = None,
```

Comment:
Can you add these to the python doc string

Reply:
👍